### ISSUE TITLE
Custom Music Disc plays my added song and Disc 11 at the same time (1.20.1)

### ISSUE THREAD

**Issue by WarrLord47** [TeamTernate/infinite-music-discs#126]
Hello, I love the idea of this project and after some messing around I finally got a song into my 1.20.1 game, but whenever I play the disc in a jukebox, it plays the added song and Disc 11 at the same time. I removed a sound overhaul resource pack I was using but it still happens. I'm new to mods and resource/datapacks so I have zero clue what's causing this.

**Comment by bradytheinventor** [comment]
Are you playing on a server? I've seen this happen before due to a laggy server connection, but it was a while ago. Any additional information you can give me about your setup, what datapacks you have installed, etc will be helpful.

**Comment by juisemonk** [comment]
Hello, I encountered tha same proplem. 
I've done one test, and I know for shure the problem is mods or datapacks they add. On my modpack Disk 11 plays alongside any of custom tracks, but there is no such proplen on pure Forge 1.20.1

**Comment by WarrLord47** [comment]
> Are you playing on a server? I've seen this happen before due to a laggy server connection, but it was a while ago. Any additional information you can give me about your setup, what datapacks you have installed, etc will be helpful.

No, I am playing singleplayer. I use a Skyrim soundpack data pack that does change the vanilla music disc but I disabled those. I've tested with all of my other datapacks off and the problem still occurs. Maybe it could be from the ambient sound mods I'm using, but none of those change the music discs.

**Comment by realBritakee** [comment]
got this be fixed cuz i have the same problem?

**Comment by LawdyLuz** [comment]
I also have the same issue , the version is V2.1 and í'm playing on miunecraft 1.20.1 over curseforge

**Comment by 1Hyrax7** [comment]
This appears to be related to Quark mod for some reason. I tested it and custom music plays without 11 in the background only when I remove Quark (+ Zeta and Oddities) from my modpack. Turning off everything in the mod doesn't fix it btw. I don't know coding so this is the best I can research. Wonder if a resource pack that disables disc 11 audio from the game would work as a fix.
